### PR TITLE
Geospatial README

### PR DESCRIPTION
**Pull request by Danwakeem**
### Description
Updated the README with documentation for the new Geospatial API.

#### Related issues

- N/A

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by Danwakeem** [comment]
@virkt25 Oh yeah, no problem thanks for reminding me!

File: README.md [hunk 1]
```diff
@@ -726,6 +727,37 @@ module.exports = function(server) {
...
+  // 'cloudantDB' is the name of Cloudant datasource created in 
+  // 'server/datasources.json' file
+  var ds = server.datasources.cloudantDB;
+
+  ds.once('connected', function() {
+    // 1. Please note `ds.connector.geoDocs()` is the correct way to call it,
+    // NOT `ds.geoDocs()`
+    // 2. This api matches the Cloudant endpoint:
+    // GET /db/_design/<design-doc>/_geo/<index-name>
+    ds.connector.viewDocs('design_doc', 'index_name', function(err, results) {
```
Comment (b-admike) [inline README.md]: did you mean `ds.connector.geoDocs` here instead of `viewDocs`?

Comment (jannyHou) [inline README.md]: Is it a typo? should be `geoDocs` I think.

File: README.md [hunk 2]
```diff
@@ -726,6 +727,37 @@ module.exports = function(server) {
...
+  // 'server/datasources.json' file
+  var ds = server.datasources.cloudantDB;
+
+  ds.once('connected', function() {
+    // 1. Please note `ds.connector.geoDocs()` is the correct way to call it,
+    // NOT `ds.geoDocs()`
+    // 2. This api matches the Cloudant endpoint:
+    // GET /db/_design/<design-doc>/_geo/<index-name>
+    ds.connector.geoDocs('design_doc', 'index_name', function(err, results) {
+      // `results` would be the data returned by quering that geospatial index
```
Comment (shimks) [inline README.md]: `querying`

**Comment by Danwakeem** [comment]
@jannyHou Oh yeah of course, happy to help out!

@shimks Good catch! It looks like that typo existed in both the `view` and `geospatial` sections so I went ahead and revised both :)

**Comment by virkt25** [comment]
@Danwakeem Thanks for the contribution! Your PR has landed.

**Comment by Danwakeem** [comment]
@virkt25 No problem and thank you too!

**Pull request merged**